### ISSUE TITLE
[next] Error 500 - Named export 'setupDevtoolsPlugin' not found.

### ISSUE THREAD

**Issue by provokcx** [nuxt-modules/i18n#1497]
### Version
@nuxtjs/i18n: 8.0.0-alpha.0-27692209.a559121
nuxt: 3.0.0-rc.8

### Nuxt configuration

Please change to `[x]` if relevant for this issue:

 - [x] Applies to a site deployed to a static server (site generated with `nuxt generate`)
 - [ ] Applies to a site deployed to a server with a Node backend

### @nuxtjs/i18n configuration

```ts
i18n: {
  locales: [
    { code: "fr", iso: "fr-FR", name: "français" },
    { code: "en", iso: "en-US", name: "english" },
  ],
  defaultLocale: "fr",
  vueI18n: {
    legacy: false,
    locale: "fr",
    messages: {
      fr: frTranslation || {},
      en: enTranslation || {},
    },
  },
},
```

### Steps to reproduce
1. `npx nuxi init nuxt-app`
2. Add `"@nuxtjs/i18n": "npm:@nuxtjs/i18n-edge"` to `package.json` `dependencies`
3. `yarn install`
4. Add `modules: ["@nuxtjs/i18n"]` and the i18n config to your `nuxt.config.ts`file
5. Open a terminal and use `yarn generate`

You should get the following error:
```
[nuxt] [request error] [unhandled] [500] Named export 'setupDevtoolsPlugin' not found.
```

### What is actually happening?
```
[nuxt] [request error] [unhandled] [500] Named export 'setupDevtoolsPlugin' not found.
```

### Related issues
https://github.com/nuxt/nuxt.js/issues/13743

**Comment by provokcx** [comment]
@kazupon 

Maybe you should add `"generate:dev": "nuxi generate playground"` to the i18n-module's `package.json` scripts to better support SSG in the future.

**Comment by kazupon** [comment]
I've just released out v8.0.0-alpha.1
In that version, this issue has already been fixed.

Thanks!
